### PR TITLE
26 finish picture

### LINKED ISSUE #26: I'm finished button that converts image to a framed picture

### LINKED ISSUE #30: Colouring: When finished, I can add my name and a title for the picture
hide palette
prevent onclick
shrink into a frame
add artist and title labels

### PR DESCRIPTION
**Pull request by Nikomus**
Fixes #26
Fixes #30

### CODE REVIEW

File: pages/doncaster-minster.tsx [hunk 1]
```diff
@@ -12,20 +42,70 @@ const Default: Colour = {
 
 export default function Game() {
   const [colour, setColour] = useState<Colour>(Default);
+  const [framed, setFramed] = useState<boolean>(false);
+  const [modalIsOpen, setIsOpen] = useState<boolean>(false);
+  const [name, setName] = useState<string>('');
+  const [title, setTitle] = useState<string>('');
 
-  const handleChange = (col: Colour) => {
+  const handleColor = (col: Colour) => {
```
Comment (katjam) [inline pages/doncaster-minster.tsx:50]: color or colour?

File: pages/shapes.tsx [hunk 2]
```diff
@@ -2,8 +2,38 @@ import { useState } from 'react';
...
+`;
+
+const Title = styled.h2`
+  font-size: 2.5rem;
+  margin: 2rem 0;
+  text-align: center;
+`;
+
+const Signature = styled.div`
+  font-family: cursive;
```
Comment (katjam) [inline pages/shapes.tsx:32]: Nice touch!

File: pages/shapes.tsx [hunk 3]
```diff
@@ -2,8 +2,38 @@ import { useState } from 'react';
 import styled from 'styled-components';
 
 import Image from '../public/colouring-images/Shapes.svg';
+import { ImageHeader } from '../components/ImageHeader';
 import { PaletteControl } from '../components/PaletteControl';
 import { Colour } from '../types';
+import { SaveModal } from '../components/SaveModal';
+import { VisuallyHidden } from '../components/VisuallyHidden';
+
```
Comment (katjam) [inline pages/shapes.tsx:10]: It feels a bit annoying that all the code has to be repeated in each image. I can't remember why it was hard to have the images as svgs that got propped into a shared component?

This design choice also makes it difficult for darts to add their own new colouring pics.

**Review by katjam — APPROVED** [review]
This looks awesome. Only niggle is that each picture needs it's own `.tsx` but that's not a new problem.

File: pages/doncaster-minster.tsx (hunk 1 above)
Comment (Nikomus) [reply pages/doncaster-minster.tsx:50]: Heck 🗽

File: pages/shapes.tsx (hunk 3 above)
Comment (Nikomus) [reply pages/shapes.tsx:10]: It is annoying. It was difficult to dynamically import an image as a React component, so I went with duplicating last time and didn't change it in this PR. I'll make an issue so it gets done another time.

**Pull request merged**